sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -107,29 +98,27 @@ public void run(C configuration, Environment environment) {
   }
 
   /**
-   * Provides a {@link ExpectedTopicConfiguration} that is generated from the values within the
-   * configuration yaml
+   * Provides a {@link TopicConfig} that is generated from the values within the configuration yaml
    *
-   * @param name the name of the topic
+   * @param key the name of the topic configuration
    * @return the configured topic configuration
    * @throws ConfigurationException if no such topic exists in the configuration
    */
-  public ExpectedTopicConfiguration getTopicConfiguration(String name)
-      throws ConfigurationException { // NOSONAR
-    if (topics.get(name) == null) {
+  public TopicConfig getTopicConfiguration(String key) throws ConfigurationException { // NOSONAR
+    if (kafkaConfiguration.getTopics().get(key) == null) {
       throw new ConfigurationException(
           String.format(
-              "Topic with name '%s' seems not to be part of the read configuration. Please check the name and configuration.",
-              name));
+              "Topic with key '%s' seems not to be part of the read configuration. Please check the name and configuration.",
+              key));
     }
-    return topics.get(name);
+    return kafkaConfiguration.getTopics().get(key);
   }
 
   /**
    * Creates a number of message listeners with the parameters given in the {@link
    * MessageListenerRegistration}.
    *
-   * <p>The depricated fields of the {@link ListenerConfig} from {@link MessageListenerRegistration}
+   * <p>The deprecated fields of the {@link ListenerConfig} from {@link MessageListenerRegistration}
```

Comment:
I don't see any deprecated fields in [`ListenerConfig`](https://github.com/SDA-SE/sda-dropwizard-commons/blob/PLP-642-remove-check-topic-configuration-and-topicana-classes-for-v-3/sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/config/ListenerConfig.java). Is this sentence still needed?
Thanks for raising awareness by fixing the typo. 🤣